models/modelTopics.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+function selectTopicData() {
+  return db.query(`SELECT * FROM topics`).then((result) => {
+    const formattedData = result.rows.map((topic) => {
+        return {
+            slug: topic.slug,
+            description: topic.description
+        }
+    })
+    return formattedData
```

Comment:
There is no need for the formatted data map. `result.rows` is already an array contains everything you need 🙂